src/api/apollo/mutations.ts

```diff
@@ -1,4 +1,4 @@
-import gql from 'graphql-tag';
```

Comment:
A general note: it would be good to establish a ts-standard or similar linter, so there's consistency in the format. That should be part of the CI as well.